src/Controllers/RegisterController.php

```diff
@@ -80,19 +83,21 @@ public function registerAction(): RedirectResponse
 
         Events::trigger('register', $user);
 
-        auth()->login($user);
+        /** @var Session $authenticator */
+        $authenticator = auth('session')->getAuthenticator();
+
+        $authenticator->login($user);
```

Comment:
```suggestion
        auth('session')->getAuthenticator()->login($user);
```

Looks like this is the only time the variable is being used so no need to create one for this.

Reply:
It is used later.
https://github.com/codeigniter4/shield/blob/b7c308d72181dadc277a6d007263c4caa7ebde4b/src/Controllers/RegisterController.php#L100

Reply:
Good deal. I missed that.